src/cuter_cerl.erl

```diff
@@ -70,7 +70,7 @@
 -type name() :: atom().
 -type fa() :: {name(), arity()}.
 -type cerl_attr_type() :: cerl_recdef() | cerl_typedef().
--type cerl_attr_spec() :: cerl_specdef().
+-type cerl_attr_spec() :: cerl_spec_form().
 
 -type cerl_recdef() :: {name(), [cerl_record_field()]} % for OTP 19.x
```

Comment:
Arguably this is unrelated to this PR, but perhaps we should take this opportunity to drop support for OTP 18.x and earlier and simplify the code of this module.  Depending on the extent of the changes, we may want to do this cleanup in this or in some separate PR.